ironfish/src/rpc/routes/chain/getTransactionStream.ts

```diff
@@ -3,40 +3,77 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 import * as yup from 'yup'
 import { Assert } from '../../../assert'
+import { AssetsValue } from '../../../blockchain/database/assets'
 import { ChainProcessor } from '../../../chainProcessor'
 import { Block } from '../../../primitives/block'
 import { BlockHeader } from '../../../primitives/blockheader'
-import { CurrencyUtils } from '../../../utils'
+import { CurrencyUtils } from '../../../utils/currency'
 import { PromiseUtils } from '../../../utils/promise'
 import { isValidIncomingViewKey } from '../../../wallet/validator'
 import { ValidationError } from '../../adapters/errors'
 import { ApiNamespace, router } from '../router'
 
 interface Note {
-  amount: string
+  assetId: string
+  assetName: string
+  value: string
   memo: string
 }
+interface Mint {
+  assetId: string
+  assetName: string
+  value: string
+}
+interface Burn {
+  assetId: string
+  assetName: string
+  value: string
+}
 
 interface Transaction {
   hash: string
   isMinersFee: boolean
   notes: Note[]
+  mints: Mint[]
+  burns: Burn[]
 }
 
 const NoteSchema = yup
   .object()
   .shape({
```

Comment:
Wasn't sure why this was mapped to `amount`, renamed to `value` for consistency. Just want to make sure we don't have any usages of these RPCs elsewhere since this is a breaking change. I searched the codebase and didn't find any, but not sure if the pool operators are using this